charts/istio-alerts/templates/service-prometheusrule.yaml

```diff
@@ -1,42 +1,50 @@
-{{ $values           := .Values }}
-{{ $release          := .Release }}
-{{ if .Values.serviceRules.enabled }}
+{{ $values     := .Values }}
+{{ $release    := .Release }}
 
+{{ if .Values.serviceRules.enabled }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ .Release.Name }}-service-rules
+  name: {{ .Release.Name }}-{{ lower .Chart.Name }}-service-rules
   annotations:
     nextdoor.com/chart: {{ .Values.chart_name }}
     nextdoor.com/source: {{ .Values.chart_source }}
 spec:
   groups:
-  - name: {{ .Release.Name }}.{{ .Release.Namespace }}.serviceRules
+  - name: {{ .Release.Namespace }}.{{ .Release.Name }}.{{ .Chart.Name }}.IstioServiceRules
     rules:
     {{ with .Values.serviceRules.http5XXMonitor }}
     {{ if .enabled }}
-    - alert: {{ $release.Name }}-5xx-Rate-Too-High
+    - alert: 5xx-Rate-Too-High
```

Comment:
I like having the name of the service in the name of the alarm I get. `dev-changelog-5xx-rate-too-high` is a lot different than `us1-nextdoor-fe-fast-5xx-rate-too-high`!

Reply:
Sorry - we have two threads here... see https://github.com/Nextdoor/k8s-charts/pull/98#discussion_r783322249 for my response.